### ISSUE TITLE
Scheduler Migrations: Query and execute commands on scheduler /v1/plan endpoint

### ISSUE THREAD

**Issue by kolloch** [mesosphere/marathon#3497]
We need to be able to query the "/v1/plan" endpoints of schedulers and execute the required commands on this endpoint. In this issue, we should have a internal scala service that provides a convenient interface with `Future`s to do so.

We should at least support

* querying the full deployment plan
* querying the status
* submitting continue, interrupt commands

Depends on #3497 for testing.

Context: We want to be able to drive scheduler upgrades manually. See [MARATHON-856](https://mesosphere.atlassian.net/browse/MARATHON-856) (private, sorry).

**Comment by kolloch** [comment]
We won't do this.
